### DISCUSSION TITLE
Ability to control notification sound and volume?

### DISCUSSION THREAD

**Discussion by noxora** [runelite/runelite#16087] [Q&A] [answered]
As far as I'm aware, the notification settings under the 'runelite' configuration menu don't allow for customizing the sound or volume notifications. I do not wish to use OS notifications because I find them more intrusive, but can't use the beep sound because it is very loud compared to my game sounds. If anyone knows where I could start reading to try and see how much work this feature would take, I'd appreciate a link

**Comment by Enriath** [comment, marked as answer]
You can already set a custom notification sound by choosing the custom option, then putting a WAV file called `notification.wav` inside `%userprofile%\.runelite` (`~/.runelite/` on Linux/OSX). You can modify this sound to be any volume you want by editing the file itself.

**Reply by noxora** [reply]
Where is the custom option? I don't see it under `Settings/Runelite/Notification Settings` I'm on windows and would be happy to do that

Edit: I've found it, it's an option under `Notification Sound` I'll implement that. Thanks!
